src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,11 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {
+    comments.map(comment => <CommentInfo comment={comment} key={comment.id} />)
```

Comment:
it`s better to divide writing map method on the several strings for better readability